src/Adapter/Configuration.php

```diff
@@ -109,7 +110,7 @@ public function get($key, $default = null)
         // if the key is multi lang related, we return an array with the value per language.
         // getInt() meaning probably getInternational()
         if (ConfigurationLegacy::isLangKey($key)) {
-            return ConfigurationLegacy::getInt($key);
+            return $this->getLocalized($key);
```

Comment:
the comment about `getInt` can be cleaned now

Reply:
You are right. :)

Reply:
Done.